lte/gateway/c/core/oai/test/s1ap_task/test_s1ap_state_converter.cpp

```diff
@@ -127,9 +127,10 @@ TEST_F(S1APStateConverterTest, S1apStateConversionExpectedEnbCount) {
 }
 
 TEST_F(S1APStateConverterTest, S1apStateConversionUeContext) {
-  ue_description_t* ue = (ue_description_t*)calloc(1, sizeof(ue_description_t));
-  ue_description_t* final_ue =
-      (ue_description_t*)calloc(1, sizeof(ue_description_t));
+  ue_description_t* ue = new ue_description_t();
+  EXPECT_TRUE(ue != nullptr);
```

Comment:
I think, the nullptr check here is not required when using `new ...` since C++ returns an instance or throws a std::bad_alloc exception.